src/utils/text.js

```diff
@@ -29,14 +29,14 @@ const isMacOS = appinfo.OS === "Darwin";
 export function formatKeyShortcut(shortcut: string): string {
   if (isMacOS) {
     return shortcut
-      .replace(/Shift\+/g, "\u21E7 ")
-      .replace(/Command\+|Cmd\+/g, "\u2318 ")
-      .replace(/CommandOrControl\+|CmdOrCtrl\+/g, "\u2318 ")
-      .replace(/Alt\+/g, "\u2325 ");
+      .replace(/Shift\+/g, "\u21E7+")
+      .replace(/Command\+|Cmd\+/g, "\u2318+")
+      .replace(/CommandOrControl\+|CmdOrCtrl\+/g, "\u2318+")
+      .replace(/Alt\+/g, "\u2325+");
```

Comment:
@Art-Vanderlay @bomsy 
The issue was about Linux and Windows. I do not have a have a Mac, but from looking at screenshot on Duck https://duckduckgo.com/?q=mac+firefox+menu&t=ffab&iar=images&iax=images&ia=images&iai=https%3A%2F%2Fevscicats.com%2Fwp-content%2Fuploads%2F2013%2F05%2F01-firefox-mac-history-menu.png it looks like it is uncommon to have a `+` on MacOS. So it should stay there probably, and only Linux and Windows part be changed.

Maybe someone with a Mac could clarify.

Reply:
@gerdneuman Yep! you're totally right, i missed that!

Reply:
@Art-Vanderlay We should probably revert the changes for macOS
```suggestion
      .replace(/Shift\+/g, "\u21E7")
      .replace(/Command\+|Cmd\+/g, "\u2318")
      .replace(/CommandOrControl\+|CmdOrCtrl\+/g, "\u2318 ")
      .replace(/Alt\+/g, "\u2325 ");
```

Reply:
No problem. Should I just create a new commit for this or revert this one?

Reply:
@Art-Vanderlay you just undo the changes above

Reply:
> @Art-Vanderlay you just undo the changes above

Ok. I made a bit of a mess of  things and then I ended up making a new [commit](https://github.com/devtools-html/debugger.html/pull/7507/commits/d5a9812464d0972ce13b231627bc43cd4dafa9fc). Will this work?

Reply:
yeah this should work!